src/logplex_logging.erl

```diff
@@ -5,7 +5,15 @@
 %% @end
 -module(logplex_logging).
 
--export([dest/2]).
+-export([dest/2
+        ,setup/0]).
+
+-include("logplex_logging.hrl").
+
+-define(SYSLOG_TAB, syslog_tab).
+-define(SYSLOG_HOST, {127, 0, 0, 1}).
+-define(SYSLOG_PORT, 514).
+-define(SYSLOG_FACILITY, local2).
```

Comment:
Should the syslog UDP connection configuration be made an application config? 

Looking it up every time is not viable; and Erlang doesn't have a concept of global variables, does it? I wonder how we can cache the said config values on repeated logging calls.